lib.php

```diff
@@ -206,24 +210,29 @@ public function get_file_results($cmid, $userid, $file, $vericite=null) {
 		//do not consult the cache if the timesubmitted is less than PLAGIARISM_VERICITE_SCORE_CACHE_IGNORE_MIN
 		$cacheTime = 60 * PLAGIARISM_VERICITE_SCORE_CACHE_MIN;
         $mycontent = null;
-        $contentscore = $DB->get_records('plagiarism_vericite_files', array('cm' => $vericite['cmid'], 'userid' => $userid, 'identifier' => $fileid), '', 'id,cm,userid,identifier,similarityscore, timeretrieved, status, timesubmitted');
+        $contentscore = $DB->get_records('plagiarism_vericite_files', array('cm' => $vericite['cmid'], 'userid' => $userid, 'identifier' => $fileid), '', 'id,cm,userid,identifier,similarityscore, preliminary, timeretrieved, status, timesubmitted');
         if (!empty($contentscore)) {
 			//there should only be 1 result
             foreach ($contentscore as $content) {
                 $mycontent = $content;
-				if(isset($mycontent->timesubmitted) && (time() - $mycontent->timesubmitted) < (60 * PLAGIARISM_VERICITE_SCORE_CACHE_IGNORE_MIN)){
```

Comment:
I noticed you removed the min recheck flag in these conditions. Have you watched the logs to make sure that we do not call our API too many times in cases with multiple submissions that are all outside of the cache limit? I believe that min recheck was a safe guard against that.